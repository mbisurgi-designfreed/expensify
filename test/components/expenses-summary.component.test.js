import React from 'react';
import { shallow } from 'enzyme';

import { ExpensesSummary } from '../../src/components/expenses-summary.component';
import expensesTotal from '../../src/selectors/expenses-total.selector';
import expenses from '../fixtures/expenses';

test('should render ExpensesSummary correctly with more than one expense', () => {
    const wrapper = shallow(<ExpensesSummary count={expenses.length} total={expensesTotal(expenses)} /> );

    expect(wrapper).toMatchSnapshot();
});

test('should render ExpensesSummary correctly with  one expense', () => {
    const wrapper = shallow(<ExpensesSummary count={[expenses[0]]} total={expensesTotal([expenses[0]])} /> );

    expect(wrapper).toMatchSnapshot();
});