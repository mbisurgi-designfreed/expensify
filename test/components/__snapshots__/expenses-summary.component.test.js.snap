// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary correctly with  one expense 1`] = `
<h1>
  Viewing 
   
  expenses
   totalling 
  $1.95
</h1>
`;

exports[`should render ExpensesSummary correctly with more than one expense 1`] = `
<h1>
  Viewing 
  3
   
  expenses
   totalling 
  $1,141.95
</h1>
`;
